aiida/tools/importexport/dbexport/__init__.py

```diff
@@ -138,241 +365,225 @@ def export(
         raise exceptions.ArchiveExportError(f"The output file '{filename}' already exists")
 
     if silent:
-        logging.disable(level=logging.CRITICAL)
+        logging.disable(logging.CRITICAL)
+
+    writer = writers[file_format](filename=filename, use_compression=use_compression)
+
+    summary(
+        writer.file_format_verbose,
+        filename,
+        include_comments=include_comments,
+        include_logs=include_logs,
+        **traversal_rules
+    )
+
+    output_data = {}
+
+    output_data['time_extract_start'] = time.time()
```

Comment:
In the context of compressed archives, `extract` is easily associated with extracting a compressed file.

I suggest using `collect` (or `query` if you prefer)
```suggestion
    output_data['time_collect_start'] = time.time()
```

Reply:
fixed in b689f5e6986c15ab18ad225e5659f1ef3bf8f9c0